### PR TITLE
upgraded dependency changes

### PR DESCRIPTION
**Pull request by maxbeatty**
fixes breaking changes introduced in #154

### CODE REVIEW

File: server/lib/schema.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ exports.testPage = Joi.object().keys({
   initHTML: mediumText,
   setup: mediumText,
   teardown: mediumText,
-  test: Joi.array().required().min(2).includes(Joi.object().required().keys({
+  test: Joi.array().required().min(2).items(Joi.object().required().keys({
```
Comment (maxbeatty) [inline server/lib/schema.js]: breaking change from [joi@6](https://github.com/hapijs/joi/issues/579)

**Comment by mathiasbynens** [comment]
Interesting that tests still passed despite these changes being necessary

**Comment by maxbeatty** [comment]
> Interesting that tests still passed despite these changes being necessary

None of the new dependencies were used in #154 because `npm-shrinkwrap.json` was not updated and that is what dictates plain `npm install` like travis does

**Comment by maxbeatty** [comment]
:shipit:

**Pull request merged**